src/Zicht/Bundle/FrameworkExtraBundle/Helper/EmbedHelper.php

```diff
@@ -122,6 +122,7 @@ public function getEmbedParams()
      * @param string $formTargetRoute
```

Comment:
Approved, maar voeg nog even een entry in de changelog toe, dan weten we waarom er weer een extra parameter nodig was